app/controllers/assessments_controller.rb

```diff
@@ -188,10 +188,14 @@ def update
     # Figure out the change
     delta = []
     typed_reported_symptoms.each do |symptom|
-      new_val = symptom.bool_value
-      old_val = assessment.reported_condition&.symptoms&.find_by(name: symptom.name)&.bool_value
-      if new_val.present? && old_val.present? && new_val != old_val
-        delta << symptom.name + '=' + (new_val ? 'Yes' : 'No')
+      new_val = symptom.value
+      old_val = assessment.reported_condition&.symptoms&.find_by(name: symptom.name)&.value
+      case symptom.type
+      when 'BoolSymptom'
+        has_changed = [true, false].include?(new_val) && [true, false].include?(old_val) && new_val != old_val
```

Comment:
I'm not clear on why this additional check is required?

Reply:
If this is protecting against the case where `old_value` would be `nil` then what about

```ruby
delta << "#{symptom.label} (\"#{old_val ? 'Yes' : 'No'}\" to \"#{new_val ? 'Yes' : 'No'}\")" if has_changed && !old_value.nil?
```

Reply:
I was thinking it wasn't really neccesary either - it's really just making sure that if somehow a bool symptom val was being attempted to be changed to something OTHER than "true" or "false" it wouldn't consider that change but I don't think that should ever be possible

Reply:
Do you mean that this still works by just checking `new_val != old_val` for BoolSymptoms?